src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/legacyMedia/index.js

```diff
@@ -0,0 +1,54 @@
+import is from 'ramda/src/is';
+
+const generateVideoBlock = block => {
+  const generatedBlock = {
+    type: 'aresMediaMetadata',
+    blockId: `urn:bbc:ares::${block.subType}:${block.id}`,
+    model: {
+      available: true,
+      blockId: block.id,
+    },
+  };
+
+  if (generatedBlock.model.format === 'video') {
+    generatedBlock.model.format = 'audio_video';
+  }
+
+  return generatedBlock;
+};
+
+const generateImageBlock = block => {
+  if (!is(String, block.imageUrl)) return {};
```

Comment:
I think this conditional is always the case for legacy media, so the rest of this function isn't used

Reply:
Removed it :+1: